### PR TITLE
Add documentation about how to debug the web application

### PR DESCRIPTION
**Pull request by jguertin**

### CODE REVIEW

File: ppr-ui/README.md [hunk 1]
```diff
@@ -31,3 +31,59 @@ npm run lint
...
 See [Configuration Reference](https://cli.vuejs.org/config/).
+
+## Debugging
+
+### VS Code with Chrome
+
+1. In VS Code install the [Debugger for Chrome](https://github.com/Microsoft/vscode-chrome-debug) plugin
+1. Launch the application: `npm run serve`
+1. In VS Code:
+    - Go to the _Debug and Run_ tab (`Shift+Command+D`)
```
Comment (WalterMoar) [inline ppr-ui/README.md]: Command? :)

**Review by WalterMoar — APPROVED** [review]
Looking forward to playing with this!

File: ppr-ui/README.md (hunk 1 above)
Comment (jguertin) [reply ppr-ui/README.md]: Oops, that's on Mac

**Review by bryan-gilbert — APPROVED** [review]
Very clear instructions.

**Pull request merged**